lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,437 @@
+    description: Indicates who performed or participated in the charge.
+    action: modify
+    example: |
+      {
+        "performer": [
+          {
+            "function": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0912",
+                  "code": "OP",
+                  "display": "Ordering Provider"
+                }
+              ],
+              "text": "Ordering Provider"
+            },
+            "actor": {
+              "reference": "Practitioner/11817978"
+            }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The value must be in the form of `CodeableConcept`</li>
+        <li>Must contain both the actor and function subfields</li>
+        <li>The value of actor must be in the form of `valueReference`</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.performer
+
+  - name: reason
+    required: 'No'
+    type: CodeableConcept
+    description: Diagnosis codes for why the charge was rendered.
+    action: modify
+    example: |
+      {
+        "reason": [
+          {
+            "extension": [
+              {
+                "valueUnsignedInt": 1,
+                "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+              }
+            ],
+            "coding": [
+              {
+                "extension": [
+                  {
+                    "valueCoding": {
+                      "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                      "code": "3693"
+                    },
+                    "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                  }
+                ],
+                "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-ICD",
+                "code": "R51"
+              }
+            ]
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the ICD proprietary Bill Codes Types systems</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.reason
+
+  - name: Replacing Extension
+    required: 'Yes'
+    type: Extension
+    description: The resource that this ChargeItem is replacing.
+    action: modify
+    example: |
+      {
+        "extension": [
+          {
+            "valueId": "1",
+            "url": "item-version"
+          }
+        ],
+        "valueReference": {
+          "reference": "ChargeItem/317370800"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/replacing"
+      }
+    note: >
+      <ul>
+        <li>Can only provide one replacing extension</li>
+        <li>The item-version sub extension is required</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
```

Comment:
it doesn't look like this URL resolves to an 'extension' field for ChargeItem in the HL7 documentation. Should this instead be: https://fhir-ehr.cerner.com/r4/StructureDefinition/replacing?_format=json ?